amethyst_renderer/src/scene.rs

```diff
@@ -65,8 +65,8 @@ impl Scene {
     /// Returns the active camera in the scene.
     ///
     /// TODO: Render to multiple viewports with possibly different cameras.
-    pub fn active_camera(&self) -> Option<&Camera> {
-        self.cameras.first()
+    pub fn active_camera(&mut self) -> Option<&mut Camera> {
```

Comment:
Why is this must?

Reply:
Mut

Reply:
I guess it's not. It was, when I put all the movement on `Camera` in the beginning. Should I remove it?

Reply:
Are we even using scene?  I think it might be dead code tbh, so ignore it

Reply:
Umm, scene.rs don't exist in the develop branch

Reply:
No sorry, the file exist but it's not used. So ignore it.